crates/serde/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-//! Serde helpers for primitive types.
+//! Alloy serde helpers for primitive types.
 
```

Comment:
this needs the regular alloy lib.rs header:

https://github.com/alloy-rs/alloy/blob/b370cf390e88f893711b1dedd90dd7654846e73a/crates/rpc-types/src/lib.rs#L6-L20

Reply:
Fixed at 👉  https://github.com/alloy-rs/alloy/pull/259/commits/3c784c4440361c1c871cdd2e5e680389c8fafb9c